ui/app/actions.js

```diff
@@ -364,6 +364,14 @@ var actions = {
   confirmChangePassword,
 
   createCancelTransaction,
+
+  //RIF
+  SHOW_DOMAINS_PAGE: 'SHOW_DOMAINS_PAGE',
```

Comment:
I think we could use a separated file for this and import those actions here from an array structure, it could be better because if tomorrow we want to diferenciate rif actions from the other actions we don't have just a comment, it's just a suggestion.

Reply:
I like this comment, you right... Let's take all RIF code to other classes/files so we can diferenciate it, it will be easier to us to mantain also.